### ISSUE TITLE
dbt-fal: decrease in speed when using dbt-fal wrapping Postgres

### ISSUE THREAD

**Issue by chamini2** [fal-ai/fal#720]
**Describe the bug**
From Slack:

> aiming at using dbt-fal[postgres]==1.3.11 in production.
> We're noticing an odd behavior when running dbt seed commands: the speed of inserts decreases drastically when updating profiles.yml to use fal. Has anyone experienced this issue?

And: 

> anecdotally dbt seed seems much slower with the adapter.  On dbt 1.3.2 dbt seed of a 6k/267-line/4-column CSV takes ~0.1s while on dbt+fal adapter it takes ~22s

<details>
closes FEA-769
</details>

**Comment by mtilda** [comment]
My team is running into the same issue when trying to implement `dbt-fal` with Postgres. We have a seed file that is ~13000 rows. Normally it loads in ~30s, but when we add `dbt-fal` it takes ~4000s (more than an hour).

These load times are not acceptable in our project, so this bug is completely blocking us from using `dbt-fal` 😔

`requirements.txt`
```
dbt-core==1.3.1
dbt-fal>=1.3.1
dbt-postgres==1.3.1
fal>=0.3.6
```

`profiles.yml`
```
default:
  target: example
  outputs:
    example:
      type: fal
      db_profile: postgres
```

**Comment by chamini2** [comment]
Thank you for letting us know, we are actively looking into this!

**Comment by chamini2** [comment]
closed by #723
